modules/price/src/Plugin/Field/FieldFormatter/PriceCalculatedFormatter.php

```diff
@@ -76,28 +84,23 @@ public static function create(ContainerInterface $container, array $configuratio
       $configuration['third_party_settings'],
       $container->get('entity_type.manager'),
       $container->get('commerce_price.number_formatter_factory'),
-      $container->get('commerce_price.chain_price_resolver')
+      $container->get('commerce_price.chain_price_resolver'),
+      $container->get('entity.manager')->getStorage('commerce_currency')
```

Comment:
entity manager is deprecated, you need the entity type manager.
We also prefer to inject the entity type manager and then get the storage in __construct()